astester.imports.js

```diff
@@ -32,15 +32,19 @@ function generateAddress() {
     }
 }
 */
-export async function local(memory) {
-  const emptyAddress = {
-    storage: new Map(),
-    contract: '',
-  };
+export function local(memory) {
   const defaultAddress = 'A12UBnqTHDQALpocVBnkPNy7y5CndUJQTLutaVDDFgMJcq5kQiKq';
+  const contractAddress = 'A12BqZEQ6sByhRLyEuf0YbQmcF2PsDdkNNG1akBJu9XcjZA1eT';
```

Comment:
where does this address come from?

Reply:
I guess it's a random value. Maybe we should add a comments